src/components/new-request/search.vue

```diff
@@ -14,17 +14,17 @@
       <!-- New BC Business flow -->
       <template v-if="isNewBcBusiness">
         <EntityType v-if="getLocation" />
-        <CompanyType v-if="getEntityTypeCd" />
-        <NumberedCompanyBullets v-if="isNumberedCompany" />
+        <CompanyType v-if="getEntityTypeCd && isNumberedEntityType" />
+        <NumberedCompanyBullets v-if="isNumberedCompany && isNumberedEntityType" />
 
-        <template v-if="isNamedCompany">
+        <template v-if="(isNamedCompany || !isNumberedEntityType) && entity_type_cd">
```

Comment:
Does this block display for a society (with the FF enabled)?

Reply:
What do you mean Sev?

Reply:
With your latest code, start a new BC business with type = Society.

Do  you see a name to request, and the Check this Name button?

Reply:
Yes.

If FF true:
![if ff true](https://github.com/bcgov/namerequest/assets/122301442/f45c461c-01a7-412f-9480-b53b894ba4ba)

If FF false:
![if ff false](https://github.com/bcgov/namerequest/assets/122301442/7b9b2cc4-563c-45d1-b915-a804a87a73a1)

Reply:
Oh, I see how you've done it...

Society is NOT set as a named company so, in my flow, I added the check for `isSociety`. In your flow, you check if it's an entity that cannot be numbered and that entity is set. I think that can be simplified (and more concise) from
```
<template v-if="(isNamedCompany || !isNumberedEntityType) && entity_type_cd">
```
to
```
<template v-if="isNamedCompany || isSociety">
```

Reply:
This doesn't seem to work Sev.

The reason is that if we're in an SP for example, the Name Input won't be shown. This is since it's neither a named company nor a society.

![sp not working](https://github.com/bcgov/namerequest/assets/122301442/66a2cb2a-d4c4-481e-9f01-d406756dd961)

Reply:
Oh, I assumed SP was automatically set as a Named company (without showing the company type component since it can never be numbered).

In amalgamations, SPs don't exist, so I didn't see this issue.

Leave it as-is then.